host-controller/src/test/java/org/jboss/as/host/controller/model/jvm/JvmOptionsBuilderUnitTestCase.java

```diff
@@ -44,13 +44,17 @@
         try {
             String vmVersionStr = WildFlySecurityManager.getPropertyPrivileged("java.specification.version", null);
             Matcher matcher = Pattern.compile("^1\\.(\\d+)$").matcher(vmVersionStr); //match 1.<number>
-            if (matcher.find()) {
-                vmVersion = Integer.valueOf(matcher.group(1));
+            if (vmVersionStr.contains(".")) {
+                if (matcher.find()) {
+                    vmVersion = Integer.valueOf(matcher.group(1));
+                } else {
+                    throw new RuntimeException("Unknown version of jvm " + vmVersionStr);
+                }
```

Comment:
You have a regex here already, why do this with half regex and half string banging?

Just use a regex of "^(?:1\\.)?(\\d+)$" instead.